x-pack/test_serverless/functional/test_suites/common/management/data_views/serverless.ts

```diff
@@ -74,9 +74,9 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
               type: DataViewType.ROLLUP,
             },
             override: true,
-          })
-          .set(ELASTIC_HTTP_VERSION_HEADER, INITIAL_REST_VERSION);
```

Comment:
Why are these request headers removed?

Reply:
whoops!  keyboard happy fingers.  on it.

Reply:
Should be good [now](https://github.com/elastic/kibana/pull/179794/commits/e6abe1444fd5cb68dcbbe4eecd81457a263e1084) boss